testsuite/cluster-test/src/instance.rs

```diff
@@ -18,9 +18,15 @@ use std::{
 };
 use tokio::{process::Command, time};
 
+#[derive(Debug, Clone)]
+pub struct ValidatorGroup {
+    pub index: u64,
```

Comment:
Should we have index: u32 here, same way `validator_group` was before? Seems like it will help to avoid a lot of `as u32` in code?